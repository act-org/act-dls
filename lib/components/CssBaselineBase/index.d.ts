/**
 * @prettier
 */
import * as React from 'react';
import { CssBaselineProps } from '@material-ui/core';
export declare type Props = CssBaselineProps;
declare const CssBaselineBase: React.FC<Props>;
export default CssBaselineBase;
//# sourceMappingURL=index.d.ts.map