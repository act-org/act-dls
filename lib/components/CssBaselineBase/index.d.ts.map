{"version":3,"file":"index.d.ts","sourceRoot":"","sources":["../../../src/components/CssBaselineBase/index.tsx"],"names":[],"mappings":"AAAA;;GAEG;AAEH,OAAO,KAAK,KAAK,MAAM,OAAO,CAAC;AAC/B,OAAO,EAAe,gBAAgB,EAAE,MAAM,mBAAmB,CAAC;AAElE,oBAAY,KAAK,GAAG,gBAAgB,CAAC;AAErC,QAAA,MAAM,eAAe,EAAE,KAAK,CAAC,EAAE,CAAC,KAAK,CAEmB,CAAC;AAEzD,eAAe,eAAe,CAAC"}