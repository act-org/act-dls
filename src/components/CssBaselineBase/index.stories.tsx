/**
 * @prettier
 */

import * as React from 'react';

import CssBaselineBase from '.';

export const Base = (): React.ReactElement<any> => (
  <CssBaselineBase>CSS Baseline</CssBaselineBase>
);

export default {
  component: CssBaselineBase,
  parameters: {
    info: {
      text: 'This is the base variant of the CssBaseline component.',
    },
  },
  title: 'Atoms/CssBaseline',
};
