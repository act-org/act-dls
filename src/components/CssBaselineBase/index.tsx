/**
 * @prettier
 */

import * as React from 'react';
import { CssBaseline, CssBaselineProps } from '@material-ui/core';

export type Props = CssBaselineProps;

const CssBaselineBase: React.FC<Props> = (
  props: Props,
): React.ReactElement<any> => <CssBaseline {...props} />;

export default CssBaselineBase;
