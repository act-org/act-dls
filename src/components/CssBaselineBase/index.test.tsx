/**
 * @prettier
 */

import * as React from 'react';

import render from '~/helpers/test/render';

import CssBaselineBase from '.';

describe('CssBaselineBase', () => {
  it('matches the snapshot', () => {
    const { container } = render(
      <CssBaselineBase>This is a test.</CssBaselineBase>,
    );

    expect(container).toMatchSnapshot();
  });
});
