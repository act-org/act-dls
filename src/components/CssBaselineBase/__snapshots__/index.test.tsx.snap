// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CssBaselineBase matches the snapshot 1`] = `
<div>
  This is a test.
</div>
`;
